### ISSUE TITLE
On Uncertainty Minimum Query Selection

### ISSUE THREAD

**Issue by FanglinLiu1** [lyuwenyu/RT-DETR#286]
May I ask the author, does uncertainty minimum query selection have anchor points?

**Comment by Zhao-Yian** [comment]
No. We did not introduce anchor points, but selected from the feature tokens output by the encoder.
